src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+
+Now on to the fun part of actually compiling gdext for Android! :)
+
+Set the environment variable CLANG_PATH to point to Android's build of clang.  Example:
+
+CLANG_PATH is used by bindgen's clang-sys dependency.  Documentation here: https://github.com/KyleMayes/clang-sys?tab=readme-ov-file#environment-variables 
+
+```bash
+#!/bin/bash
+
+export CLANG_PATH = $"[android-cli-directory]/[android-cli-version]/ndk/[ndk-version]/toolchains/llvm/prebuilt/[host-machine-os]/bin/clang"
+```
+
+Then set the ``CARGO_TARGET_[target-triple-here]_LINKER`` to point to the android linker for the android triple you are targetting.  You need to compile your gdext library for each android triple individually.  You can find possible targets by running:
+
+```
+rustup target list
+```
+
+You can find the linkers in the android cli directory at:
+
+```
+$"[android-cli-directory]/[android-cli-version]/ndk/[ndk-version]/toolchains/llvm/prebuilt/[host-machine-os]/bin/[target-triple-here][android-version]"
+```
+
+As of writing this, the tested tuples are:
+
+| Tuple                       | Environment Variable                            | Godot Arch     | Gdextension Config            |
+| --------------------------- | ----------------------------------------------- | -------------- | ----------------------------- |
+| ``aarch64-linux-android``   | ``CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER``   | ``arm64``      | ``android.debug.arm64``       |
+| ``x86_64-linux-android``    | ``CARGO_TARGET_X86_64_LINUX_ANDROID_LINKER``    | ``x86_64``     | ``android.debug.x86_64``      |
+| ``armv7-linux-androideabi`` | ``CARGO_TARGET_ARMV7_LINUX_ANDROID_LINKER``     | ``arm32``      | ``android.debug.armeabi-v7a`` |
+| ``i686-linux-android``      | ``CARGO_TARGET_I686_LINUX_ANDROID_LINKER``      | ``x86_32``     | ``android.debug.x86``         |
+
+Notice how the environment variables are in all caps and the triple's "-" is replaced with "_"
+
+Make sure to add all of the triples you want to support to the ``rustup`` via:
+
+```
+rustup target add [android-triple-here]
+```
+
+Example:
+
+```
+rustup target add aarch64-linux-android
+```
+
+Putting it all together, here is an example compiling for ``aarch64-linux-android``.  This is also probably the most common Android target as of the writing of this.
+
+Assuming 2 things:
+
+1. android cli is installed in my HOME folder
+2. Godot is still relying on android ndk version: 23.2.8568313.  Check here: https://github.com/godotengine/godot/blob/4a0160241fd0c1e874e297f6b08676cf0761e5e8/platform/android/java/app/config.gradle#L14
+3. The downloaded android cli version is: 11076708_latest (update this to be the version you downloaded)
+4. This is being ran on linux.  Change the ``linux-x86_64`` folder in ``CLANG_PATH`` and ``CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER`` to be your host machine's operating system.
+5. You are targetting android version 34
+
+```bash
+#!/bin/bash
+
+rustup target add aarch64-linux-android
```

Comment:
Introduce this code block with surrounding text. If it's not a script, omit `#!/bin/bash`.